### DISCUSSION TITLE
The first use of WikiDocs ?!

### DISCUSSION THREAD

**Discussion by ics01** [Zavy86/WikiDocs#71] [Show and tell]
We are very proud to be in the earlier (the first ?!?!) user of WikiDocs IRL ! @Zavy86 🥇 

This wiki motor is really incredible ! 
- Very simple to install
- Very simple to use
- Perfectly designed, as Italian do at their best !

Warm congratulations to Manuel Z. and all the team ! 👍 👍 👍 

(What we've done is here https://scanip.ics-informatique.com)

**Comment by Zavy86** [comment]
merci beaucoup! 😀

thank you for sharing your experience!

🍻

**Comment by ics01** [comment]
Onestamente, il tuo lavoro è superbo ed efficiente.
Non ho particolari competenze su Internet, ma se posso aiutarti 
(traduzione?), non esitare!

Etienne

------ Message d'origine ------
De: "Manuel Zavatta" ***@***.***>
À: "Zavy86/WikiDocs" ***@***.***>
Cc : "ics01" ***@***.***>; "Author" 
***@***.***>
Envoyé : 25/08/2023 08:29:00
Objet : Re: [Zavy86/WikiDocs] The first use of WikiDocs ?! (Discussion 
#71)
